src/geojson/geojson.ml

```diff
@@ -101,16 +124,17 @@ module Make (J : Geojson_intf.Json) = struct
 
       let of_json json = parse_by_type json parse_coords typ
 
-      let to_json positions =
+      let to_json (positions, bbx) =
         J.obj
           [
             ("type", J.string typ);
+            ("bbox", Option.(if is_some bbx then J.array J.float (get bbx) else J.null));
             ("coordinates", J.array Position.to_json positions);
           ]
     end
 
     module LineString = struct
-      type t = Position.t array
+      type t = Position.t array * float array option
```

Comment:
Let's change all of these to records if that's okay. I think it'll be clearers i.e. 

```ocaml
type t = {
  coordinates : Position.t array;
  bbox : float array option
}
```